lms/djangoapps/learner_dashboard/rest_api/v1/urls.py

```diff
@@ -0,0 +1,15 @@
+"""
+URLs for the V1 of the Learner Dashboard API.
+"""
+from django.conf.urls import url
+
+from .views import ProgramListView
+
+app_name = 'v1'
```

Comment:
Is `app_name` important for the functionality to work correctly? The data entered here seems to vary from app-to-app in the LMS and I'm trying to understand if it is even important.

Sometimes I see it just as `v1` or `v0`. I've seen it as the name of the djangoapp (`app_name = 'certificates'`), and I have also seen it a more formal declaration that looks like the path to the djangoapp (`app_name = 'lms.djangoapps.grades'`). 

Does any of it matter?